operator/src/main/java/oracle/kubernetes/operator/helpers/PodHelper.java

```diff
@@ -32,10 +32,13 @@
 import oracle.kubernetes.operator.work.Packet;
 import oracle.kubernetes.operator.work.Step;
 import oracle.kubernetes.weblogic.domain.model.ServerSpec;
+import oracle.kubernetes.weblogic.domain.model.Shutdown;
 
 public class PodHelper {
   private static final LoggingFacade LOGGER = LoggingFactory.getLogger("Operator", "Operator");
 
+  private static final int DEFAULT_ADDITIONAL_DELETE_TIME = 10;
```

Comment:
Why the 10 second fudge factor?

Reply:
Would this cause a shutdown timeout of 1 second to actually become 11 seconds?  This seems significant for folks like me that are often just trying to get rid of a domain as fast as possible

Reply:
I'd like to punt on resolving this too...  (See the comment below).  I'd like to, if you'll agree, work in a separate PR on getting the operator to connect via REST to shutdown the server.  Then, when that has worked, we won't need to fudge factor for the WLST initialization time.

Reply:
OK.